scripts/dev.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# local development start script to prepare dev related env
+
+SCRIPTS_DIR=$(dirname "${BASH_SOURCE[0]}")
+
+bash "$SCRIPTS_DIR/set-pusher-suffix.sh"
```

Comment:
Why have this split up into separate files if this is all it does? Let's just put it all in one file if we can.

Reply:
The idea for `dev.sh` is to host a list of scripts we run before starting local development

If that's undesired, I'll just delete it and use `set-pusher-suffix.sh` directly